config/admins.txt

```diff
@@ -6,128 +6,128 @@
-Gun Hog = Game Master
-KorPhaeron = Game Master
-razharas = Game Master
-Lordpidey = Game Master
-Niknakflak = Game Master
-rolan7 = Game Master
-quarxink = Game Master
-adrix89 = Game Master
-tle = Game Master
-xsi = Game Master
-scaredofshadows = Game Master
-neofite = Game Master
-trubblebass = Game Master
-mport2004 = Game Master
-deuryn = Game Master
-agouri = Game Master
-errorage = Game Master
-superxpdude = Game Master
-petethegoat = Game Master
-korphaeron = Game Master
-nodrak = Game Master
-carnwennan = Game Master
-ikarrus = Game Master
-cheridan = Game Master
-giacomand = Game Master
-rockdtben = Game Master
-sieve = Game Master
-aranclanos = Game Master
-intigracy = Game Master
-dumpdavidson = Game Master
-kazeespada = Game Master
-malkevin = Game Master
-incoming = Game Master
-demas = Game Master
-fleure = Game Master
-ricotez = Game Master
-misterperson = Game Master
-crimsonvision = Game Master
-iamgoofball = Game Master
-zelacks = Game Master
-androidsfv = Game Master
-miggles = Game Master
-jordie0608 = Game Master
-s0ldi3rkr4s0 = Game Master
-ergovisavi = Game Master
-vistapowa = Game Master
-miauw62 = Game Master
-kazeespada = Game Master
-rumia29 = Game Master
-bobylein = Game Master
-sirbayer = Game Master
-hornygranny = Game Master
-yota = Game Master
-firecage = Game Master
-donkieyo = Game Master
-argoneus = Game Master
-paprka = Game Master
-cookingboy3 = Game Master
-limeliz = Game Master
-steelpoint = Game Master
-phil235 = Game Master
-CorruptComputer = Game Master
-xxnoob = Game Master
-tkdrg = Game Master
-Cuboos = Game Master
-thunder12345 = Game Master
-wjohnston = Game Master
-mandurrh = Game Master
-thurgatar = Game Master
-xerux = Game Master
-dannno = Game Master
-lo6a4evskiy = Game Master
-vekter = Game Master
-Ahammer18 = Game Master
-ACCount12 = Game Master
-fayrik = Game Master
-shadowlight213 = Game Master
-drovidicorv = Game Master
-Dunc = Game Master
-MMMiracles = Game Master
-bear1ake = Game Master
-CoreOverload = Game Master
-Jalleo = Game Master
-ChangelingRain = Game Master
-FoxPMcCloud = Game Master
-Xhuis = Game Master
-Astralenigma = Game Master
-Tokiko1 = Game Master
-SuperSayu = Game Master
-Lzimann = Game Master
-As334 = Game Master
-neersighted = Game Master
-Swankcookie = Game Master
-Ressler = Game Master
-Folix = Game Master
-Bawhoppennn = Game Master
-Anturke = Host
-Lumipharon = Game Master
-bgobandit = Game Master
-coiax = Game Master
-RandomMarine = Game Master
-PKPenguin321 = Game Master
-TechnoAlchemist = Game Master
-Aloraydrel = Game Master
-Quiltyquilty = Game Master
-SnipeDragon = Game Master
-Fjeld = Game Master
-kevinz000 = Game Master
-Tacolizard = Game Master
-TrustyGun = Game Master
-Cyberboss = Game Master
-PJB3005 = Game Master
-Sweaterkittens = Game Master
-Feemjmeem = Game Master
-JStheguy = Game Master
-excessiveuseofcobby = Game Master
-Plizzard = Game Master
+###############################################################################################

```

Comment:
why is almost every line changed in this file

Reply:
I just copied text from github.

Reply:
screwed up line endings
git checkout upstream/master config/admins.txt